tests/integration/defs/accuracy/references/cnn_dailymail.yaml

```
@@ -230,6 +233,10 @@ mistralai/Mixtral-8x22B-v0.1:
     accuracy: 25.519
   - quant_algo: W8A16
     accuracy: 27.427
+microsoft/Phi-4-mini-instruct:
```

YAMLlint (1.37.1)

[error] duplication of key "microsoft/Phi-4-mini-instruct" in mapping

(key-duplicates)